pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
     "Topic :: Scientific/Engineering",
 ]
 dependencies = [
-    "gym >=0.21.0",
+    "gymnasium >=0.28.1",
```

Comment:
@perrin-isir `pyproject.toml` has moved to the `tools/pypi` subdirectory now (to prevent users tempted to `pip install .` in the repo, since it doesn't work that way :sweat_smile:). You will probably see that when rebasing this branch to solve the conflict.